<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Options</class>
 <widget class="QDialog" name="Options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>563</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>220</y>
     <width>301</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>Параметры линии</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>23</y>
      <width>251</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="2" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Подпись:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="currentSeriesName"/>
     </item>
     <item row="3" column="1">
      <spacer name="verticalSpacer_6">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Цвет:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="currentSeriesColor_CB">
       <item>
        <property name="text">
         <string>красный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>зеленый</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>синий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>черный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>оранжевый</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="1">
      <spacer name="verticalSpacer_8">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Тип линии:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="currentSeriesLineType_CB">
       <item>
        <property name="text">
         <string>сплошная</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>точки</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>пунктир</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>пунктир с точкой</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="7" column="1">
      <spacer name="verticalSpacer_7">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Толщина:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="seriesWidth"/>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QComboBox" name="currentSeriesChoise_CB"/>
     </item>
     <item row="1" column="0">
      <spacer name="verticalSpacer_9">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>301</width>
     <height>371</height>
    </rect>
   </property>
   <property name="title">
    <string>Параметры графика</string>
   </property>
   <widget class="QCheckBox" name="axisX_CB_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Шкала по оси Х</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="axisX_sign_CB_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Подпись по оси Х</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="axisX_sign_text_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="axisY_sign_CB_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Подпись по оси Y</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="axisY_CB_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Шкала по оси Y</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="auto_minY_CB_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Авто min(Y)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="auto_maxY_CB_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Авто max(Y)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_interval_range_2">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>300</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Watch unterval (sec. -1 - inf):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_interval_update_2">
    <property name="geometry">
     <rect>
      <x>66</x>
      <y>330</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Update interval (ms.):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="interval_range_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>330</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="interval_upd_ms_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>330</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="axisY_sign_text_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="legend_align">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>240</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Снизу</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Сверху</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Справа</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Слева</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>240</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Легенда:</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="test_label">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>90</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="testlabel">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>150</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>test label</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>490</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>Ок</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="applyButton">
      <property name="text">
       <string>Применить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="closeButton">
      <property name="text">
       <string>Закрыть</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>261</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Все доступные серии</string>
   </property>
   <widget class="QListWidget" name="allSeries">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>241</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>281</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Расположение графиков</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="chartLayot_CB">
      <item>
       <property name="text">
        <string>Один график</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Вертикально</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Горизонтально</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Сетка</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>258</width>
        <height>18</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Количество стобцов:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="chartColNumber_spin">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>5</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Количество строк:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="chartRowNumber_spin">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>5</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="createLayot_button">
      <property name="text">
       <string>Создать расположение</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
